Jenkinsfile_CNP

```diff
@@ -14,7 +14,7 @@ def product = "bar"
 def app = "api"
 
 withPipeline(type, product, app) {
-    installCharts()
+    //installCharts()
```

Comment:
@AnilDhawaleGitHub - Please remove this.

Reply:
Its Removed